src/AbstractPhpdocToTypeDeclarationFixer.php

```diff
@@ -95,6 +97,10 @@ protected function createConfigurationDefinition(): FixerConfigurationResolverIn
                 ->setAllowedTypes(['bool'])
                 ->setDefault(\PHP_VERSION_ID >= 8_00_00)
                 ->getOption(),
+            (new FixerOptionBuilder('map_types', 'Map of custom types, eg template types from PHPStan.'))
```

Comment:
IMHO it should be `types_map`, as `map_types` looks like an intent (or boolean flag, like the other options).

Reply:
In general, I believe that PRs introducing changes in public contract of Fixer/rules shouldn't be merged without review.

Reply:
`types_map` vs `map_types` - open for both. no opinion

public contract - explicitly, [it is not that case](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/doc/rules/function_notation/phpdoc_to_param_type.rst#this-rule-is-experimental). Abstract parent class is [internal](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/src/AbstractPhpdocToTypeDeclarationFixer.php#L32) as well

Reply:
#8293